consensus/state.go

```diff
@@ -530,10 +530,11 @@ func (s State) PartialSigHash(txn types.Transaction, cf types.CoveredFields) typ
 	return h.Sum()
 }
 
-// TransactionsCommitment returns the commitment hash covering the transactions
-// that comprise a child block.
-func (s State) TransactionsCommitment(txns []types.Transaction, v2txns []types.V2Transaction) types.Hash256 {
+// Commitment computes the commitment hash for a child block with the given
+// transactions and miner address.
+func (s State) Commitment(minerAddr types.Address, txns []types.Transaction, v2txns []types.V2Transaction) types.Hash256 {
 	var acc blake2b.Accumulator
+	acc.AddLeaf(hashAll(uint8(0), "commitment", s.v2ReplayPrefix(), types.Hash256(hashAll(s)), minerAddr))
 	for _, txn := range txns {
```

Comment:
Do we need to commit to the miner payouts?

Reply:
good question! v2 blocks only have a single payout, and its value is fully determined by the block height, so there is no need to commit to it.

Reply:
I did a double take because it’s still a slice. Sneaky https://github.com/SiaFoundation/core/blob/498585cae108f0532f333bbd85e5b74ee0df7471/consensus/validation.go#L58